vic/vic_run/include/vic_physical_constants.h

```diff
@@ -108,6 +108,8 @@
 #define CONST_CPFW 4.188e3  /**< specific heat of fresh h2o ~ J/kg/K */
 #define CONST_CPFWICE 4.2e3  /**< specific heat of fresh h2o ~ J/kg/K */
 #define CONST_CPICE 2.11727e3  /**< specific heat of fresh ice ~ J/kg/K */
+/**< volumetric heats */
+#define CONST_VCPICE_WQ (CONST_CPICE * CONST_RHOFW)  /**< heat capacity of fresh ice per volume of water equivalent ~ J/m^3/K */
```

Comment:
OK - water equivalent - I'll just answer my own question. I'll go ahead and merge.